### PR TITLE
Added COVID-19 questionnaire (symptoms, clinical history, smoking, fever)

### PR DESCRIPTION
**Pull request by danielmlow**
See issues "Set up more elegantly" and "Checkboxes"

### CODE REVIEW

File: covid19/items/covid19_clinical_history [hunk 1]
```diff
@@ -0,0 +1,110 @@
+{
+    "@context": [ "https://raw.githubusercontent.com/ReproNim/reproschema/master/contexts/generic",
+    "https://raw.githubusercontent.com/sensein/covid19/master/covid19/covid19_context"
```
Comment (satra) [inline covid19/items/covid19_clinical_history:3]: you are not using this context anywhere, so just reduce this list to the one url. this is true for all the items below.

**Comment by satra** [comment]
@danielmlow - this PR includes a lot of files that should not be here (.idea, .DS_Store, etc.,.) please remove them from git history, and add them to `.gitignore`

File: covid19/items/covid19_clinical_history (hunk 1 above)
Comment (danielmlow) [reply covid19/items/covid19_clinical_history:3]: Reduce the list to this, right?
```
"@context": ["https://raw.githubusercontent.com/sensein/covid19/master/covid19/covid19_context"]

Comment (satra) [reply covid19/items/covid19_clinical_history:3]: it should reduce it to this to the other url

```
"@context": "https://raw.githubusercontent.com/ReproNim/reproschema/master/contexts/generic",
```

**Comment by satra** [comment]
@danielmlow - to clean up this PR, you may want to remove the extra files, `squash` your git commits and `rebase` on top of master, and then `force` push to your repo.

**Pull request closed without merging**